p2p/config/config.go

```diff
@@ -56,6 +58,7 @@ type Config struct {
 	SwarmConfig           SwarmConfig   `mapstructure:"swarm"`
 	BufferSize            int           `mapstructure:"buffer-size"`
 	MsgSizeLimit          int           `mapstructure:"msg-size-limit"` // in bytes
+	MaxGossipRoutines     int           `mapstructure:"max-gossip-routines"`
```

Comment:
user shouldn't be responsible/able to configure stuff like that

Reply:
most users won't touch these config values, but the config is useful so we can test different values in, e.g., different devnets/test networks or other test environments